src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEPropertySymbol.cs

```diff
@@ -687,8 +687,7 @@ internal override bool MustCallMethodsDirectly
                 var ordinal = i - 1;
                 bool isBad;
 
-                // https://github.com/dotnet/roslyn/issues/29821: handle extra annotations
```

Comment:
I assume you checked there are no references left to this issue ID

Reply:
Yes

---
In reply to: [303279434](https://github.com/dotnet/roslyn/pull/34916#discussion_r303279434) [](ancestors = 303279434)